modules/library/render/src/test/java/org/geotools/renderer/lite/AbstractLabelLineTest.java

```diff
@@ -67,6 +65,23 @@ public void setUp() throws Exception {
         bounds = featureSource.getBounds();
         bounds.expandBy(1, 1);
     }
+    
+    protected Style loadParametricStyle(Object loader, String sldFilename, String... parameters) throws IOException {
+        StyleFactory factory = CommonFactoryFinder.getStyleFactory(null);
```

Comment:
Ah! it will help indeed test different variants without having to repeat the styles, thanks.